main_spec.tex

```diff
@@ -163,6 +163,24 @@ \subsubsection{\textbf{SHMEM\_FETCH}}\label{subsec:shmem_fetch}
 \subsubsection{\textbf{SHMEM\_SET}}\label{subsec:shmem_set}
 \input{content/shmem_set.tex}
 
```

Comment:
Could you fix the ordering of shmem_add and shmem_fadd so they appear together and with the fetching version first (CSWAP, SWAP, FINC, INC, FADD, ADD, ...)?  Also, should the bitwise ops appear before fetch/set?